package.json

```diff
@@ -33,7 +33,7 @@
     "media-typer": "",
     "morgan": "",
     "node-hipchat": "",
-    "protagonist": "0.15.0",
+    "protagonist": "0.19.0",
```

Comment:
Sure, but Heroku wouldn't manually update the dependencies. It would require the slug to be re-built, which only happens if we commit something. So perhaps it makes sense to update this explicitly here.